frontend/device/devicelistener.lua

```diff
@@ -26,6 +26,10 @@ end
 function DeviceListener:onToggleNightMode()
     local night_mode = G_reader_settings:isTrue("night_mode")
     Screen:toggleNightMode()
+    -- Make sure CRe will bypass the call cache
+    if self.ui and self.ui.document and self.ui.document.provider == "crengine" then
```

Comment:
(You could just check `if self.ui and self.ui.rolling then` - a bit cleaner but less explicite, so ok :)

Reply:
Good to knows, thanks ;).